include/openenclave/bits/properties.h

```diff
@@ -70,6 +70,10 @@ typedef struct _oe_enclave_properties_header
     UUID, HEAP_SIZE, STACK_SIZE, FLAGS, VERSION, DESCRIPTION)
 #endif
 
+/**
+ * @cond DEV
```

Comment:
For my understanding, why are we choosing not to document this function?

Reply:
This change is in reference to Issue [#2359](https://github.com/openenclave/openenclave/issues/2359). 
Quoting __oe_sgx_set_minimum_crl_tcb_issue_date() to be a private API that shouldn't show up in a public document.